packages/v4/patternfly-docs/content/design-guidelines/components/card/card.md

```diff
@@ -53,56 +53,68 @@ PatternFly offers 2 types of border treatments for cards:
  
 Use cards in a catalog view to present users with a series of selectable items, typically arranged into a grid.
  
-Note that this view uses a multi-select card so that bulk actions can be performed from the toolbar on a group of cards at once. For more information on cards used in a catalog or card view, see the [card view design guidelines](/demos/card-view/design-guidelines).
+Note that this view uses cards with a multi-select option so that bulk actions can be performed from the toolbar on a group of cards at once. For more information on cards used in a catalog or card view, see the [card view design guidelines](/demos/card-view/design-guidelines).
  
-<img src="./img/multi-select-cards.png" alt="cards in a catalog view " width="1210" />
+<img src="./img/cards-multiselect-catalog.png" alt="example of cards in a catalog view " width="1210" />
  
 ### Cards in primary-detail views
  
-Use selectable cards when you want to present an overview of a set of objects and then examine the details. Here, the details of different projects can be displayed in a side-by-side primary-detail view.
+Use clickable cards when you want to present an overview of a set of objects and then examine the details. Here, the details of different projects can be displayed in a side-by-side primary-detail view.
  
-<img src="./img/single-select-cards.png" alt="cards in a side-by-side primary-detail view " width="1225" />
+<img src="./img/cards-primarydetail.png" alt="example of cards in a side-by-side primary-detail view " width="1500" />
  
 ### Cards in dashboards
  
 Use cards in a dashboard view to visualize information. Each card should display a different and specific piece of information, such as a bar chart, area chart, or other data type.
  
-<img src="./img/dashboard.png" alt="card view in a dashboard with charts" width="1223"/>
+<img src="./img/dashboard.png" alt="example of card view in a dashboard with charts" width="1223"/>
```

Comment:
are the cards here using the correct background? Like is that not the styling for card on white background instead of card on grey background? Could be wrong just wanted to double check!!